src/stylesheets/theme/_sds-theme-custom-styles.scss

```diff
@@ -42,6 +42,7 @@ SDS THEME CUSTOM STYLES
 // Layouts
 // -------------------------------------
 @import '../layouts/subheader';
+@import '../layouts/page';
```

Comment:
adding this could be misunderstood/misuse by some developers. Developers could think that page specific styles could be added here - All css should be broken down in to components and add it to their corresponding folder

Reply:
I moved the sds-page css to a component file, hopefully this is more clear.